pkgs/development/python-modules/scikit-optimize/default.nix

```diff
@@ -28,13 +31,9 @@ buildPythonPackage rec {
 
   checkInputs = [
     pytest
```

Comment:
`pytest` is implicit in `pytestCheckHook`, you can remove it from here.

Reply:
Thx. Done.